data/Iyanu1396.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "IyanuOluwa",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer | Javascript ",
+  "avatar": "https://avatars.githubusercontent.com/u/108562866?s=400&u=63b79f7eb10da91446c3af03ee453029795889f1&v=4",
+"links": [
+    {
+      "name": "Connect with me on Twitter",
+      "url": "https://twitter.com/iyanu_codes",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://t.co/LEuD34NGk2",
```

Comment:
You might want to add your GitHub profile link directly, rather than a Twitter redirect, IMO.